core/amber/src/main/scala/Engine/Operators/PythonUDF/FieldTypeConverter.scala

```diff
@@ -0,0 +1,30 @@
+package Engine.Operators.PythonUDF
+
+import Engine.Common.AmberField.FieldType
+
+object FieldTypeConverter {
+  def convertOne(javaField: FieldTypeInJava): FieldType.Value = javaField match {
```

Comment:
maybe name it convertType. Both the function here can be called convertType and you can overload the parameters

Reply:
Done